cavern/src/intTest/java/org/opencadc/cavern/TransferRunnerTest.java

```diff
@@ -208,6 +246,14 @@ public void testTransferNegotiation21() {
             Assert.assertFalse("cookieTLS", plist.contains(cookieTLS));
             Assert.assertFalse("token", plist.contains(token));
             Assert.assertFalse("tokenTLS", plist.contains(tokenTLS));
+            
+            // check the file content
```

Comment:
Not sure it is useful to do the download here since this test is about expected combinations of protocols and security methods (negotiation only)